src/pages/ProductDetail/ProductDetail.jsx

```diff
@@ -17,6 +19,14 @@ function ProductDetail() {
     return <Loader />;
   }
 
+  const stockAvailable = (stock) => {
+    return quantity > 0 && quantity <= stock ? true : false;
```

Comment:
the ternary can be omitted and the condition can be returned instead:

```suggestion
    return quantity > 0 && quantity <= stock;
```